src/Components/nav/Nav.css

```diff
@@ -0,0 +1,20 @@
+.active {
+  font-weight: bold;
+}
+
+.list {
+  list-style: none;
+  margin-right: 2rem;
+}
+
+.flexed-list {
+  position: fixed;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
If I remove these two, and replace then with `margin: auto`, there seems to be no visible change. What am I missing or is it your preferred approach?